packages/react/src/primitives/Menu/MenuButton.tsx

```diff
@@ -28,11 +28,18 @@ export const MenuButton = React.forwardRef<
     ref
   ) => {
     const { propStyles, nonStyleProps } = useStyles(rest, style);
+    size = size || 'default';
+    variation = variation || 'default';
 
     return (
       <button
         ref={ref}
-        className={classNames(ComponentClassNames.Button, className)}
+        className={classNames(
+          ComponentClassNames.Button,
+          `${ComponentClassNames.Button}--size-${size}`,
```

Comment:
How do people feel about removing 'size' and 'variation' from the classnames? So instead this would be

```js
`${ComponentClassNames.Button}--${size}`
```